3723/1/cli.rb

```diff
@@ -0,0 +1,26 @@
+# rubocop:disable Metrics/MethodLength, Lint/ShadowingOuterLocalVariable
```

Comment:
AFAIK we don't disable cops for this task

Reply:
Done